django/contrib/admin/options.py

```diff
@@ -555,7 +556,12 @@ def wrapper(*args, **kwargs):
             url(r'^add/$', wrap(self.add_view), name='%s_%s_add' % info),
             url(r'^(.+)/history/$', wrap(self.history_view), name='%s_%s_history' % info),
             url(r'^(.+)/delete/$', wrap(self.delete_view), name='%s_%s_delete' % info),
-            url(r'^(.+)/$', wrap(self.change_view), name='%s_%s_change' % info),
+            url(r'^(.+)/edit/$', wrap(self.change_view), name='%s_%s_change' % info),
+            # For backwards compat
```

Comment:
No, just an extension of that comment so that future readers can easily tell when the change was made. If someone sees that note while working on Django 10.0, they know it's probably safe to consider removing it.